apps/server/src/modules/deletion/api/uc/deletion-request.uc.spec.ts

```diff
@@ -175,21 +244,30 @@ describe(DeletionRequestUc.name, () => {
 			it('should call eventBus.publish', async () => {
 				const { deletionRequest } = setup();
 
-				deletionRequestService.findAllItemsToExecute.mockResolvedValueOnce([deletionRequest]);
-
 				await uc.executeDeletionRequests();
 
 				expect(eventBus.publish).toHaveBeenCalledWith(
 					new UserDeletedEvent(deletionRequest.id, deletionRequest.targetRefId)
 				);
 			});
+
+			it('should work with a delay of 0', async () => {
+				setup(true);
+
+				await uc.executeDeletionRequests();
+
+				expect(deletionRequestService.markDeletionRequestAsFailed).not.toHaveBeenCalled();
```

Comment:
somehow the test szenario does not make sense to me with the given description of "should work..."

Reply:
this was needed to for the coverage to test also the last line
return Promise.resolve();